Source/Core/Core/Movie.h

```diff
@@ -10,17 +10,13 @@
 #include <string>
 
 #include "Common/CommonTypes.h"
+#include "Core/HW/WiimoteCommon/DataReport.h"
```

Comment:
These still seem like the necessary data structures can be forward declarations.

Reply:
👍